cura/PrinterOutputDevice.py

```diff
@@ -74,6 +111,30 @@ def setTargetBedTemperature(self, temperature):
         self._target_bed_temperature = temperature
         self.targetBedTemperatureChanged.emit()
 
+    ## Time the print has been printing.
+    #  Note that timeTotal - timeElapsed should give time remaining.
+    @pyqtProperty(float, notify = timeElapsedChanged)
+    def timeElapsed(self):
+        return self._time_elapsed
+
+    ## Total time of the print
+    #  Note that timeTotal - timeElapsed should give time remaining.
+    @pyqtProperty(float, notify=timeTotalChanged)
+    def timeTotal(self):
+        return self._time_total
+
+    @pyqtSlot(float)
+    def setTimeTotal(self, new_total):
```

Comment:
Out of curiosity, why are these exposed to QML? Is there ever a reason to set this from QML?